# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

int32 patch_id
bool external_observations
bool left
bool left_for_nav
bool right
bool switch_possible
bool strong_switch
float32 x
float32 y